sdk/typescript/src/serialization/base64.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Buffer } from 'buffer';
+
+// TODO: Buffer is not supported in browser environments
+export class Base64DataBuffer {
```

Comment:
```
// TODO: Buffer is not supported in browser environments
```
Sorry this was an outdated comment. With `import { Buffer } from 'buffer';`, this is working in browsers. 

I introduced this class because in many case we we receive a base64-encoded string, and we want to convert it to bytes for signing. In the interface, I would like it to be explicit that the input should be a base64 encoded string rather than a regular string.

Reply:
makes sense